### ISSUE TITLE
Tooltips Position Offset

### ISSUE THREAD

**Issue by 299792458ms** [Alexays/Waybar#3342]
Are tooltips by default offset to the right of the center of the corresponding module now? I just changed my colorscheme and nothing else really. I tested all tooltips and all of them has this behaviour.

Anyway I wanted to know if could be and issue from my end(maybe something related to gtk themes?) and if someone has run into the same issue(yes I did search the issue section). Or maybe there is a way to style the tooltip to set an offset? It really seems like I messed something up so if anyone knows what could it be I would really appreciate it.

Below are the pictures, their relevant config sections and some info

![screenshot2](https://github.com/Alexays/Waybar/assets/152849182/f698c69c-8146-435b-98ff-d0a2b2d57327)
![2024-06-08-183854_hyprshot](https://github.com/Alexays/Waybar/assets/152849182/5ef117f8-997e-42ac-87b6-f720998addb2)

```
// config.jsonc

   "clock": {
        "tooltip-format": "<tt><small>{calendar}</small></tt>",
        "format-alt": "{:%Y-%m-%d}",
        "calendar": {
            "mode"          : "month",
            "on-scroll"     : 1,
            "format": {
                "months":     "<span color='#CDD6F4'><b>{}</b></span>",
                "days":       "<span color='#CDD6F4'><b>{}</b></span>",
                "weeks":      "<span color='#F9E2AF'><b>W{}</b></span>",
                "weekdays":   "<span color='#F9E2AF'><b>{}</b></span>",
                "today":      "<span color='#F38BA8'><b><u>{}</u></b></span>"
            }
        },
        "tooltip": true
    }
```
```
/* style.css*/

tooltip {
    color: #d8dee9;
    background-color: @bg;
    text-shadow: none;
    border: solid 2px @fg;
}
```
Versions: 
- waybar v0.10.3
- hyprland-git Tag: v0.40.0-161-gaf5f2492

EDIT: This only happens with this specific version of Hyprland from the AUR.

**Comment by cooukiez** [comment]
same issue here, i noticed that all tooltips have an offset not just the calendar widget

**Comment by cooukiez** [comment]
I'm not sure if this has something to do with this commit #3195, because it changed some mouse entering behavior in src/AModule.cpp. But I don't really know the codebase, so this is just an assumption.
There are also other commits which are about mouse behavior / tooltip format #3213 but I'm not sure if they're relevant.

**Comment by dantearaujo1** [comment]
iSame issue here

**Comment by itsmenewbie03** [comment]
no updates on this yet?

**Comment by 299792458ms** [comment]
> no updates on this yet?

Is is fixed in Hyprland v41.2 Arch Extra Repo.
